src/test/ABDKMath64x64.sol

```diff
@@ -0,0 +1,880 @@
+// SPDX-License-Identifier: BSD-4-Clause
+/*
+ * ABDK Math 64.64 Smart Contract Library.  Copyright © 2019 by ABDK Consulting.
+ * Author: Mikhail Vladimirov <mikhail.vladimirov@gmail.com>
+ */
+pragma solidity ^0.8.0;
+
+/**
+ * Smart contract library of mathematical functions operating with signed
+ * 64.64-bit fixed point numbers.  Signed 64.64-bit fixed point number is
+ * basically a simple fraction whose numerator is signed 128-bit integer and
+ * denominator is 2^64.  As long as denominator is always the same, there is no
+ * need to store it, thus in Solidity signed 64.64-bit fixed point numbers are
+ * represented by int128 type holding only the numerator.
+ */
+library ABDKMath64x64 {
```

Comment:
including this library in tests is odd to me. we don't need to test whether or not the math library works right, so we shouldn't need to do our own math. Instead, the tests we do should have known values and make sure that the known values produce the results we expect when we use a normal calculator that can do floating point math correctly. If you're dead set on doing math inside some tests, then use the math library that we're using in the code. Otherwise you're just asking for headaches using a completely different math library that could have its own subtle bugs.

Reply:
Okay great! That library was a little hard to work with, so I am super happy removing it.... I just wasn't sure what the best or standard way is to test Solidity math. You had a comment that said, you know, do different math, don't just copy the math, to test the results. Maybe I took the comment a little too far I guess. 

I'll be interested to see if you think what is there now is enough, regarding checks.